charts/generic/pod-cpu-hog/engine.yaml

```diff
@@ -29,7 +29,6 @@ spec:
               #verify the resources the app has been launched with
             - name: CPU_CORES
```

Comment:
Can you add one comment `Container name where chaos has to be injected` above Target_container